cypress/e2e/tests/migration/applicationinventory/analysis/custom-rules.test.ts

```diff
@@ -130,6 +130,23 @@ describe(["@tier2"], "Bug MTA-2015: Custom Rules in analyses", function () {
         app.verifyFileNotValidXML();
     });
 
+    it("Python custom rules file analysis", function () {
+        const app = new Analysis(
+            getRandomApplicationData("python-app-custom-rules", {
+                sourceData: this.appData["python-demo-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["python_demo_application"])
+        );
+        tackleTestappName = app.name;
+        Application.open();
+        applications.push(app);
+        app.create();
+        app.manageCredentials(sourceCredential.name, mavenCredential.name);
```

Comment:
Hi @midays, I noticed that both credentials are being used, which isn't a concern but just for your information, since the analysis mode is set to source-code. Is there any particular reason?

Reply:
removed